jobs/build/release/Jenkinsfile

```diff
@@ -136,6 +136,11 @@ node {
                     arch: arch,
                 )
             }
+            stage("advisory image list") {
+                filename = "${params.NAME}-image-list.txt"
+                sh "elliott advisory-images -a ${advisory} > ${filename}"
```

Comment:
`commonlib.shell`
And it would likely be a good idea to retry or allow failures.

Reply:
done ;D